### PR TITLE
[WIP] Plan app

### PR DESCRIPTION
**Pull request by bhaettasch**
Integrating schedule, room and event planning. Attendance will be moved here, too.

### CODE REVIEW

File: plan/templates/plan/index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{% extends "dashboard/action.html" %}{% load bootstrap3 %}
+
+{% block title%}Plan{% endblock %}
+
+{% block content_dashboard_view %}
+    <link rel='stylesheet' href='//cdnjs.cloudflare.com/ajax/libs/fullcalendar/3.3.1/fullcalendar.min.css' />
```
Comment (exploide) [inline plan/templates/plan/index.html]: until now, we used bower for frontend dependencies

Comment (Miterion) [inline plan/templates/plan/index.html]: This is a work in progress addition to test the calendar. Actually the whole file should not be in here

File: plan/dashboard_urls.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+
+from . import dashboard_views
+
+
+app_name = 'plan'
+urlpatterns = [
```
Comment (jlauinger) [inline plan/dashboard_urls.py]: urlpatterns = []

File: plan/dashboard_views.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+
+
+class PlanAppMixin(DashboardAppMixin):
+    app_name_verbose = "Plan"
+    app_name = 'plan'
+
+    @property
+    def sidebar_links(self):
+        return [
```
Comment (jlauinger) [inline plan/dashboard_views.py]: return []

File: plan/admin.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+                           name=_('Teilnehmerliste'))
+
+
+@admin.register(Event)
+class EventAdmin(admin.ModelAdmin):
+    list_display = ['timeslot', 'room', 'tutorgroup']
+
+
+@admin.register(Booking)
+class EventAdmin(admin.ModelAdmin):
```
Comment (jlauinger) [inline plan/admin.py]: Shouldn't this be `BookingAdmin`?

Comment (bhaettasch) [inline plan/admin.py]: Fixed with afdd35a5509b2198526ae21c2d144693387125f2

**Pull request closed without merging**